docs/sections/contributor_guide/developer_setup.rst

```diff
@@ -1,94 +1,60 @@
-|                     |                                                            |
-|                     | docstrings with `docformatter`_,                           |
-|                     | and ``.jsonschema`` documents with `jq`_                   |
-+---------------------+------------------------------------------------------------+
-| ``make lint``       | Lint with `pylint`_                                        |
-|                     |                                                            |
-+---------------------+------------------------------------------------------------+
-| ``make typecheck``  | Typecheck with `mypy`_                                     |
-+---------------------+------------------------------------------------------------+
-| ``make unittest``   | Run unit tests and report coverage with `pytest`_ and      |
-|                     | `coverage`_                                                |
-+---------------------+------------------------------------------------------------+
-| ``make test``       | Equivalent to                                              |
-|                     | ``make lint && make typecheck && make unittest``           |
-|                     |                                                            |
-|                     | Checks defined CLI scripts                                 |
-+---------------------+------------------------------------------------------------+
-
-Note that ``make format`` is never run automatically, to avoid reformatting under-development code in a way that might surprise the developer. A useful development idiom is to periodically run ``make format && make test`` to perform a full code-quality sweep through the code. An additional check is run by the CI for unformatted code, ``make format`` must be run, and then changes from ``make format`` must be committed before CI will pass.
-
-The ``make test`` command is also automatically executed when ``conda`` builds a ``uwtools`` package, so it is important to periodically run these tests during development and, crucially, before merging changes, to ensure that the tests will pass when CI builds the ``workflow-tools`` code.
-
-
-The order of the targets above is intentional, and possibly useful:
-
-   * ``make format`` will complain about certain kinds of syntax errors that would cause all the remaining code-quality tools to fail (and may change line numbers reported by other tools, if it ran after them).
-   * ``make lint`` provides a good first check for obvious errors and anti-patterns in the code.
-   * ``make typecheck`` offers a more nuanced look at interfaces between functions, methods, etc. and may spot issues that would cause ``make unittest`` to fail.
+----------------------------------
 
+A development shell makes availabe several code-formatting and quality checkers, which should be periodically run during the development process. See :doc:`Code Quality <code_quality>` for full details.
```

Comment:
Most of the content moved to the _Code Quality_ page, but I added this note with a link to that page.